public/app/features/alerting/unified/components/folder-bulk-actions/FolderBulkActionsButton.tsx

```diff
@@ -0,0 +1,94 @@
+import { useState } from 'react';
+
+import { config, locationService } from '@grafana/runtime';
+import { Dropdown, Menu } from '@grafana/ui';
+import { t } from 'app/core/internationalization';
+import MoreButton from 'app/features/alerting/unified/components/MoreButton';
+
+import { alertingFolderActionsApi } from '../../api/alertingFolderActionsApi';
+import { FolderAction, useFolderAbility } from '../../hooks/useAbilities';
+import { useFolder } from '../../hooks/useFolder';
+import { createRelativeUrl } from '../../utils/url';
+
+import { DeleteModal } from './DeleteModal';
+import { FolderActionMenuItem } from './MenuItemPauseFolder';
+interface Props {
+  folderUID: string;
+}
+
+export const FolderBulkActionsButton = ({ folderUID }: Props) => {
+  const [pauseSupported, pauseAllowed] = useFolderAbility(FolderAction.Pause);
+  const canPause = pauseSupported && pauseAllowed && false; // lets disable pause for now
+  const [deleteSupported, deleteAllowed] = useFolderAbility(FolderAction.Delete);
+  const canDelete = deleteSupported && deleteAllowed;
+  const [pauseFolder, updateState] = alertingFolderActionsApi.endpoints.pauseFolder.useMutation();
+  const [unpauseFolder, unpauseState] = alertingFolderActionsApi.endpoints.unpauseFolder.useMutation();
+  const [deleteGrafanaRulesFromFolder, deleteState] =
+    alertingFolderActionsApi.endpoints.deleteGrafanaRulesFromFolder.useMutation();
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
+  const folderName = useFolder(folderUID).folder?.title || 'unknown folder';
+  const listView2Enabled = config.featureToggles.alertingListViewV2 ?? false;
+  const view = listView2Enabled ? 'list' : 'grouped';
+
+  if (!canPause && !canDelete) {
+    return null;
+  }
+
+  const onConfirmDelete = async () => {
+    await deleteGrafanaRulesFromFolder({ namespace: folderUID }).unwrap();
+    redirectToListView(view);
+  };
+
+  const menuItems = (
+    <>
+      {canPause && (
+        <>
+          <FolderActionMenuItem
+            folderUID={folderUID}
+            action="pause"
+            executeAction={async (folderUID) => {
+              await pauseFolder({ namespace: folderUID }).unwrap();
+              redirectToListView(view);
+            }}
+            isLoading={updateState.isLoading}
+          />
+          <FolderActionMenuItem
+            folderUID={folderUID}
+            action="unpause"
+            executeAction={async (folderUID) => {
+              await unpauseFolder({ namespace: folderUID }).unwrap();
+              redirectToListView(view);
+            }}
+            isLoading={unpauseState.isLoading}
+          />
+        </>
+      )}
+      {canDelete && (
+        <Menu.Item
+          label={t('alerting.folder-bulk-actions.delete.button.label', 'Delete rules')}
+          icon="trash-alt"
+          onClick={() => setIsDeleteModalOpen(true)}
+          disabled={deleteState.isLoading}
+        />
+      )}
+    </>
+  );
+
+  return (
```

Comment:
I have a slight concern regarding the screen real estate used by this button. If the expectation is that the Folder Actions are not used frequently, the current button size appears quite large. Could we consider alternatives that require less space

Reply:
I've just pushed a commit using the vertical elipsis icon instead: 
<img width="397" alt="Screenshot 2025-05-06 at 16 37 04" src="https://github.com/user-attachments/assets/01fdd87b-590e-4de7-9286-af65a78bc2b0" />